ritual/src/processor.rs

```diff
@@ -227,7 +227,13 @@ fn build_crate(data: &mut ProcessorData<'_>) -> Result<()> {
         } else {
             command.current_dir(path);
         }
-        run_command(&mut command)?;
+        run_command(&mut command).or(match cargo_cmd {
+            &"test" => {
```

Comment:
This can be improved by a paremeter. Since X cannot be started in a Docker env, every GUI test fails.